app/controllers/api/physical_servers_controller.rb

```diff
@@ -52,9 +52,20 @@ def refresh_resource(type, id, _data = nil)
       end
     end
 
+    # Process apply config pattern operation for single and multi-resources
+    #
+    # Even the request for multi-resources isn't completed successfully, return a HTTP Status 200
+    def apply_config_pattern_resource(type, id, data)
+      ensure_customization_script_exists(data["pattern_id"])
+      change_resource_state(:apply_config_pattern, type, id, data)
+    rescue StandardError => err
```

Comment:
This should not only be catching `StandardError`s but all errors, which is consistent with the rest of the API

Reply:
Should I change into change_resource method too?

Reply:
@igorrran Yes that would be great 👍

Reply:
@jntullo I have changed the code to be catching all errors, but the codeclimate have been warning about this. Could you take a look please?